packages/components/psammead-most-read/src/index.jsx

```diff
@@ -2,39 +2,36 @@ import React from 'react';
 import styled from 'styled-components';
 import { shape, string, oneOf, arrayOf, node } from 'prop-types';
 import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import SectionLabel from '@bbc/psammead-section-label';
 import MostReadList from './List/index';
-import MostReadTitle from './Title/index';
 import MostReadRank from './Rank/index';
 import { MostReadLink, MostReadItemWrapper } from './Item/index';
 
-const StyledSection = styled.section.attrs(props => ({
-  'aria-labelledby': props.labelId,
+const MostReadSection = styled.section.attrs(props => ({
   role: 'region',
+  'aria-labelledby': props.labelId,
 }))``;
 
-const MostReadSection = ({ labelId, children }) => (
```

Comment:
You might need to retain this logic and have it exporting `SectionLabel` from `@bbc/psammead-section-label` so that we can use it in the AMP implementation.

Reply:
addressed this in this commit: https://github.com/bbc/psammead/pull/2952/files/a545b44af1dd1360b471c8bbb20146681c7cf254..09e07e8f00b6ddbbaea95afb9be3a36d2eb6c83b